sdcm/cluster.py

```diff
@@ -174,14 +174,11 @@ class Setup(object):
 
     @classmethod
     def test_id(cls):
-        if not cls._test_id:
-            cls._test_id = uuid.uuid4()
```

Comment:
Why are you remove this code ? what about the case we aren't using hydra ? 
also we are using this value when configuring the logs directory we might want to change that too..

Reply:
see `Setup.logdir()` where `Setup.test_id()` is being called, and that happens before get to the `Tester.__init__()`

maybe it's bad practice to configure the logging on import, but otherwise we loose some logs of things happening on import